pkg/chains/signing.go

```diff
@@ -48,6 +49,21 @@ type TaskRunSigner struct {
 	SecretPath        string
 }
 
+type Verifier interface {
```

Comment:
could we move all verification code to `pkg/chains/verify.go`? might just make it easier to find

Reply:
For sure! So to verify I believe all we should need are:
- `cfg config.Config`
- `kc kubernetes.Interface`, which we'll use to actually read the ConfigMap into `cfg`

We can use `kc` to get the ConfigMap object from the cluster, and then we can pass the data into [NewConfigFromConfigMap](https://github.com/tektoncd/chains/blob/main/pkg/config/config.go#L202) to get `cfg`. Once we have `cfg`, I believe that's everything we need for verification.

I don't think we need the `PipelineClientset` for anything, just the kubernetes Client should be sufficient.

Does this make sense?